apps/web/src/components/(dashboard)/layout/profile-dropdown.tsx

```diff
@@ -67,6 +69,15 @@ export const ProfileDropdown = ({ user }: ProfileDropdownProps) => {
           </Link>
         </DropdownMenuItem>
 
+        {isUserAdmin && (
+          <DropdownMenuItem asChild>
+            <Link href="/admin" className="cursor-pointer">
```

Comment:
Aren't links by default already applied with the "cursor-pointer" style?

Reply:
followed the style from the existing code. all the links have that className

Reply:
Dropdown items lack cursor styling, this is intentional :)